README.md

```diff
@@ -6,168 +6,171 @@
-        git checkout development
-    ```
+Open a command window and navigate to the path where you would like to clone the repository.
+Use the following command to begin a clone:
 
-4. Setting up the project locally.
+```
+git clone https://github.com/<your github username>/social-platform-donut-frontend.git
+```
 
-    ```shell
-    npm install
-    ```
+Once the command completes you will have a new folder containing the cloned repository. We can validate this by running the “dir” command. We can then move into that directory using the "cd " command.
 
-5. Setting up the local database instance 
-      
-      You need to have a MongoDB atlas account to create local database.
-      Check out [this link](https://www.youtube.com/watch?v=KKyag6t98g8) to setup MongoDB atlas.
-      The mongodb URL should be pasted in `credential.js` files.
+![image of command line](https://i.imgur.com/QsQy4Sg.png)
 
+**Setting up a remote upstream**
 
+The next step is to set up a remote. Remotes simply represent paths or URLs to other versions of your repository. In our case, as we cloned from our fork on GitHub a default remote will have been setup for us called origin. This origin allows us to push and pull code from our forked repository hosted on GitHub. We can list the currently configured remotes on our machine using the “git remote” command.
 
-6. run this command to run on local
+Pushing and pulling from your own fork is very useful and this will be how you will work with the project most often. However, when working on that code, you’ll want to be starting from the most recent version of the code from the main donut-frontend repository. That code may have been updated and changed since you first made your fork. In order to get access to that latest code, we’ll setup a second remote which points to the main donut-frontend repository. We will not have commit rights there, so we cannot push changes, however, we will be able to fetch the latest commits that have occurred.
 
-    ```shell
-    nodemon server.js
-    ```
-    or
-    ```shell
-    node server.js
-    ```
+To create a new remote we use the “git remote add” command, passing in a name for the new remote and the URL as arguments. In our case since we want the second remote to point towards the original donut-frontend repository we will use the follwing command. It's possible to name the remote anything you like, but the convention is to name it "upstream".
 
-HURRAY! you have done it just start contributing to the project.
+```
+git remote add upstream https://github.com/codeuino/social-platform-donut-frontend.git
+```
 
-Now open your browser and type the link `http://localhost:3000` to continue viewing project.
+**Installing the dependencies and running our local version of donut-frontend**
 
-Setting Up Database
-------------
+The next step is to install the required dependencies to our newly created donut-frontend app. This could be done by running the following command in the terminal.
```

Comment:
@ksraj123 I haven't seen any open source projects mentioning dependencies in Readme

Reply:
@Rupeshiya I agree, I just mentioned it because we have that in the backend readme too. Maybe we could consider removing it from there as well or just add a tech stack section which tells about the major dependencies like express mongoose etc along with their versions.

Reply:
yeah, that's why I removed there too. Thanks